### PR TITLE
Use recent Reason fast pipe syntax in README

### PR DESCRIPTION
**Pull request by MoOx**
Tell me if I should adjust something else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,23 +58,23 @@ Future.make(resolve => resolve("hi"))
...
+let future_B = future_A->Future.map(n => n + 1);
 
 
 future_A
-|. Future.get(n => Js.log(n)); /* logs: 99 */
+->Future.get(n => Js.log(n)); /* logs: 99 */
 
 future_B
-|. Future.get(n => Js.log(n)); /* logs: 100 */
+->Future.get(n => Js.log(n)); /* logs: 100 */
```
Comment (scull7) [inline README.md]: Perhaps these examples should be taken from two lines to one?
```reason
future_A->Future.get(n => Js.log(n)); /* logs: 99 */
future_B->Future.get(n => Js.log(n)); /* logs: 100 */
```

**Review by scull7 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gilbert) [reply README.md]: I agree

Comment (MoOx) [reply README.md]: Adjusted

**Comment by gilbert** [comment]
Thank you! 😄

**Pull request merged**